contracts/dao-dao-core/src/contract.rs

```diff
@@ -174,6 +184,21 @@ pub fn execute_pause(
         .add_attribute("until", until.to_string()))
 }
 
+pub fn execute_unpause(deps: DepsMut, sender: Addr) -> Result<Response, ContractError> {
+    let admin = ADMIN.load(deps.storage)?;
+
+    // Only the admin may call this method.
+    if sender != admin {
+        return Err(ContractError::Unauthorized {});
+    }
+
+    PAUSED.remove(deps.storage);
```

Comment:
```suggestion
    PAUSED.remove(deps.storage)?;
```

i think this line is missing an unwrap?

Reply:
oh nvm